docs/pages/admin-guides/management/guides/awsoidc-integration-rds.mdx

```diff
@@ -0,0 +1,289 @@
+---
+title: AWS RDS Enrollment Wizard
+description: Enroll AWS RDS databases with your Teleport cluster using an enrollment wizard.
+---
+
+The AWS RDS enrollment wizard is available in your Teleport cluster Web UI as
+part of the [AWS OIDC integration](./awsoidc-integration.mdx).
+
+The enrollment wizard configures AWS IAM resources and deploys a Teleport
+Database Service instance on ECS Fargate to proxy connections to AWS RDS
+databases.
+
+The enrollment wizard can be used to enroll RDS databases in a Teleport cluster.
+It can also be useful as an example serverless deployment of the Teleport database
+service on ECS.
+
+## Prerequisites
+
+1. A running Teleport cluster
+1. An AWS account and permissions to create IAM Identity Providers and roles
+
+## Step 1/2. Configure RBAC
+
+The RDS enrollment wizard is a part of the AWS OIDC integration.
+You will need the following allow rules in one of your Teleport roles.
+These are available by default in the preset `editor` role:
+
+```yaml
+kind: role
+version: v7
+metadata:
+  name: example
+spec:
+  allow:
+    rules:
+    - resources:
+      - integration
+      verbs:
+      - create
+      - update
+      - list
+      - read
+```
+
+## Step 2/2. Visit the Web UI
+
+The RDS enrollment wizard is available in the "Add new resource" panel of the
+Teleport Web UI:
+
+![Screenshot of RDS enrollment wizard tile](../../../../img/database-access/enrollment-wizard/resource-panel-rds.png)
+
+The Teleport Web UI walks you through the steps to set up an AWS OIDC
+integration (if you don't have one already), configure IAM permissions, create
+an ECS deployment in your AWS account, and run a database connection test.
+
+## How it works
```

Comment:
I really want to be detailed in the explanation of how this all works, because a recurring customer complaint is that after they go through this enrollment wizard they don't know how to "do it for real" with terraform since the wizard does it all magically for them.

I'm not sure how we should present that info in a detailed explanation if you want just a short summary at the top.
How about I make subsections under "How it works"?
I'll push a change to illustrate what I mean.

Reply:
updated with subsections and tocDepth: 3 so the page is easier to navigate: 2b316271e5fd9496db4c106da93de4e45005f1c6

e.g 
![image](https://github.com/user-attachments/assets/df9e4710-f3d5-48a9-8ab2-d5529c5efd67)